### ISSUE TITLE
Checkboxes have the same colour and white outline as radio buttons

### ISSUE THREAD

**Issue by rachelhbd** [350org/ak_intl_v3#394]
### Context
Builds on/dependent on #395 

With this petition with a white background, the square checkboxes are only just visible:
https://act.350.org/sign/rachel-test-petition/

Can they have a line around the checkboxes to match the radio buttons above them, as well as the same background shade before they are selected?

![Image](https://user-images.githubusercontent.com/22962348/193334413-4956ca87-f9ff-4525-9fef-c277f6cb48ab.png)

In general, we want all radio buttons and checkboxes to look visually consistent, including on embedded pages eg. https://350.org/rachel-test-petition-embed/?akid=.2483206.ZpvPZ4

### This will be done when
- [x] Checkboxes on all page types match the formatting of the radio buttons on this page https://act.350.org/sign/rachel-test-petition/ (in terms of background colour and outline when unselected)
- [x] Checkboxes on embedded pages are also consistent eg. https://350.org/rachel-test-petition-embed/?akid=.2483206.ZpvPZ4

**Comment by rachelhbd** [comment]
Same question on css files as #395!

**Comment by sukhada** [comment]
This is actually not ready for testing because we still need to merge in this new PR: https://gitlab.com/350/web-tools/baseline/-/merge_requests/59 that adds the compiled files to the codebase. Once it's merged in, this should be good to test!

**Comment by rachelhbd** [comment]
hey @rezziemaven would it be possible to merge this in in gitlab?

**Comment by rezziemaven** [comment]
Oh hey @rachelhbd! Actually I didn't need to merge this in again because I was able to compile the SASS file to CSS after all using Buddy. So we can close this one.

**Comment by sukhada** [comment]
@rachelhbd sorry, forgot to update you, but this should be ready to QA now. @rezziemaven thanks for your help here!

**Comment by rachelhbd** [comment]
Ah cool, thanks both - I've spot checked this on a few browers and on mobile and as a checkbox goes it looks really nice 😀 ! I don't think it needs deeper QA than that so let's include in the October release

**Comment by rachelhbd** [comment]
Looks good in Develop!
